data/mods/Magiclysm/items/comestibles.json

```diff
@@ -95,10 +95,14 @@
   {
     "id": "impure_meat",
     "type": "COMESTIBLE",
-    "copy-from": "mutant_meat",
-    "extend": { "flags": [ "NUTRIENT_OVERRIDE" ] },
+    "copy-from": "meat",
```

Comment:
There's no point copy-fromming anything else if you're just copying mutant meat's stats. Change only the value you need (calories), don't switch everything around like this.

Reply:
Ok, sorry